config/services/dataplane_v1beta1_openstackdataplaneservice_ovn.yaml

```diff
@@ -7,3 +7,6 @@ spec:
   playbook: osp.edpm.ovn
   configMaps:
   - ovncontroller-config
+  secrets:
+  - neutron-external-config
```

Comment:
we may want to sync names used by operators for EDP.

Reply:
I will use <instance-name>-<service-name>-external-config for all secrets / configmaps for neutron. I think external is a better name for EDP than "compute" (as in nova patch).